### ISSUE TITLE
grunt compass and webfont collision

### ISSUE THREAD

**Issue by phouri** [sapegin/grunt-webfont#210]
It sounds a bit weird, but I've got this problem:

I run webfonts first and put the results in a directory, then I proceed to use grunt-contrib-comass, what happens is: webfonts says - no need to run, fonts are cached, and grunt-contrib-compass is acting as if it runs webfonts again (Generated font with 1 glyph), and I fail to get the resulting compiled sass. 

The issue isn't present in 0.4.8 so I guess it has something to do with the new cache?

**Comment by badunk** [comment]
I also get issues when it detects there's nothing to do.  Perhaps it is not properly exiting the grunt task?  I see error messages such as this:

```
Running "sprite:release" (sprite) task
Files "dist/css/sprite.css", "dist/assets/icons.png" created.

Running "webfont:release" (webfont) task
Font icons wasn’t changed since last run.

Running "concurrent:release" (concurrent) task
    Font icons with 11 glyphs created.
        
        Running "autoprefixer:release" (autoprefixer) task
        File dist/css/sprite.css created.
        
        Running "htmlbuild:release" (htmlbuild) task
        >> File dist/index.html created !
        
        Running "htmlmin:release" (htmlmin) task
        Minified dist/index.html 2.19 kB → 2 kB
        
        Running "cacheBust:assets" (cacheBust) task
        The file dist/index.html was busted!
        
        Done, without errors.

```
I happen to be using LESS, but I think that's actually irrelevant?

**Comment by sapegin** [comment]
@badunk Which grunt-webfont version do you use? This probably was fixed in v0.5.1.

**Comment by badunk** [comment]
Ah yes, I manually reverted to 0.5.0 temporarily.  I see your fix in 0.5.1:

https://github.com/sapegin/grunt-webfont/blob/master/tasks/webfont.js#L142

Upgrading fixed for me
